packages/headless/src/features/search/search-actions.ts

```diff
@@ -110,6 +114,12 @@ export const executeSearch = createAsyncThunk<
   'search/executeSearch',
   async (searchAction: TransitiveSearchAction, config) => {
     const state = config.getState();
+
+    if (
+      (state as StateNeededByAnswerAPI).generatedAnswer.answerConfigurationId
```

Comment:
You could use a type guard function here I think

Reply:
I will remove all changes here